### PR TITLE
Fix #4740

### LINKED ISSUE #4740: show osm notes later/higher zoom (configurable)
I can overlay osm notes. 
But they are shown much too early and clutter the map.
Personaly i want them at the 500m level or even later.

### PR DESCRIPTION
**Pull request by AlexSyt**

### CODE REVIEW

File: OsmAnd/res/values/strings.xml [hunk 1]
```diff
@@ -9,6 +9,8 @@
          3. All your modified/created strings are in the top of the file (to make easier find what\'s translated).
     PLEASE: Have a look at http://code.google.com/p/osmand/wiki/UIConsistency, it may really improve your and our work  :-)  Thx - Hardy
     -->
+    <string name="show_closed_notes">Show closed notes</string>
+    <string name="switch_osm_notes_visibility_desc">Show/Hide OSM Notes on the map.</string>
```
Comment (HolgerJeromin) [inline OsmAnd/res/values/strings.xml]: Am i wrong or is this string definition not matching the code?

File: OsmAnd/src/net/osmand/plus/OsmandSettings.java [hunk 2]
```diff
@@ -1155,6 +1155,8 @@ protected Integer getDefaultValue() {
 
 	// this value string is synchronized with settings_pref.xml preference name
 	public final OsmandPreference<Boolean> SHOW_OSM_BUGS = new BooleanPreference("show_osm_bugs", false).makeGlobal();
+	public final CommonPreference<Boolean> SHOW_CLOSED_OSM_BUGS = new BooleanPreference("show_closed_osm_bugs", false).makeGlobal();
+	public final CommonPreference<Integer> SHOW_OSM_BUGS_MIN_ZOOM = new IntPreference("show_osm_bugs_min_zoom", 8).makeGlobal();
```
Comment (HolgerJeromin) [inline OsmAnd/src/net/osmand/plus/OsmandSettings.java]: Settings code here without matching string.xml?

File: OsmAnd/res/values/strings.xml (hunk 1 above)
Comment (AlexSyt) [reply OsmAnd/res/values/strings.xml]: These are the strings for UI.

File: OsmAnd/src/net/osmand/plus/OsmandSettings.java (hunk 2 above)
Comment (AlexSyt) [reply OsmAnd/src/net/osmand/plus/OsmandSettings.java]: These settings should not match anything. I'll add an empty line to separate them.

Comment (HolgerJeromin) [reply OsmAnd/src/net/osmand/plus/OsmandSettings.java]: I thought the first parameter is used to find the appropriate localization (and therefor has to match name in string.xml).
But ok. I am no android expert

**Pull request merged**